Common subdirectories: librsvg-2.26.0.b/data and librsvg-2.26.0.c/data
Common subdirectories: librsvg-2.26.0.b/doc and librsvg-2.26.0.c/doc
Common subdirectories: librsvg-2.26.0.b/gdk-pixbuf-loader and librsvg-2.26.0.c/gdk-pixbuf-loader
Common subdirectories: librsvg-2.26.0.b/gtk-engine and librsvg-2.26.0.c/gtk-engine
Common subdirectories: librsvg-2.26.0.b/moz-plugin and librsvg-2.26.0.c/moz-plugin
diff -u librsvg-2.26.0.b/rsvg-base.c librsvg-2.26.0.c/rsvg-base.c
--- librsvg-2.26.0.b/rsvg-base.c	2014-01-15 20:56:01.992092644 -0500
+++ librsvg-2.26.0.c/rsvg-base.c	2014-01-15 21:19:50.755724506 -0500
@@ -610,6 +610,7 @@
 	    goto fallback;
 
 	xml_parser = xmlCreatePushParserCtxt (&rsvgSAXHandlerStruct, ctx, NULL, 0, NULL);
+	xml_parser->options |= XML_PARSE_NONET;
 
 	buffer = _rsvg_xml_input_buffer_new_from_stream (stream, NULL /* cancellable */, XML_CHAR_ENCODING_NONE, &err);
 	g_object_unref (stream);
@@ -1156,6 +1157,7 @@
     if (handle->priv->ctxt == NULL) {
         handle->priv->ctxt = xmlCreatePushParserCtxt (&rsvgSAXHandlerStruct, handle, NULL, 0,
                                                       rsvg_handle_get_base_uri (handle));
+	handle->priv->ctxt->options |= XML_PARSE_NONET;
 
         /* if false, external entities work, but internal ones don't. if true, internal entities
            work, but external ones don't. favor internal entities, in order to not cause a
Common subdirectories: librsvg-2.26.0.b/tests and librsvg-2.26.0.c/tests
